main.smd

```diff
@@ -5,9 +5,15 @@ description: Documents with scientific intelligence
 title: Stencila
 ---
 
+```` css raw
+ 
+
+
+````
+
```

Comment:
This empty block is showing up at the top, just behind the header when you expand the cards, just wandering if its required for a hack of some kind, or can it be removed